unified-codes/apps/web/src/components/containers/app/AppFooter.tsx

```diff
@@ -41,26 +44,16 @@ const getStyles = (theme: ITheme) => ({
       fontSize: 14,
       textTransform: 'uppercase',
     },
-  },
-  paper: {
-    alignItems: 'center',
-    backgroundColor: theme.palette.background.footer,
-    display: 'flex',
-    flexDirection: 'column' as flexDirection,
-    fontFamily: 'roboto',
-    ' & div': {
-      flex: '0 1 0%',
-    },
-  },
+  }
 });
 
 export type Footer = React.FunctionComponent<FooterProps>;
 
-const _Footer: Footer = ({ classes }) => {
+const Footer: Footer = ({ classes }) => {
```

Comment:
Still think a bit of presentational logic can be extracted from here (and logos should be moved into app rather than library)... but think that can be covered by #153?